remote-workstations/new-agent-vm/Install-PCoIPAgent.sh

```diff
@@ -308,6 +314,31 @@ do
     fi
 done
 
+# Install and setup the Sumo Collector
+echo "-->Install SumoLogic collector"
+mkdir /tmp/sumo
+wget "https://collectors.sumologic.com/rest/download/linux/64" -O /tmp/sumo/SumoCollector.sh && sudo chmod +x /tmp/sumo/SumoCollector.sh
+wget "$ARTIFACT_LOCATION/user.properties$SAAS_TOKEN" -O /tmp/sumo/user.properties
+wget "$ARTIFACT_LOCATION/sumo-agent-vm-linux.json$SAAS_TOKEN" -O /tmp/sumo/sumo-agent-vm-linux.json
+JSON_FILE=/tmp/sumo/sumo-agent-vm-linux.json
+echo "Attemtping to set sumo collector ID to: " "$COLLECTOR_ID"
+sed -i s/collectorID/"$COLLECTOR_ID"/ /tmp/sumo/user.properties
+sed -i 's|syncsourceFile|'$JSON_FILE'|' /tmp/sumo/user.properties
+
+sudo /tmp/sumo/SumoCollector.sh -q -varfile user.properties
```

Comment:
Are we assuming that this will always succeed? Do we care if it fails?

Reply:
I don't think this would impact user at all if it fails. Just harder to do debug.

Reply:
If we don't really care if it succeeds or fails, then I think we should at least check for failure and ensure that we don't execute the `service` calls in that case.

Reply:
I'll create separate issue if we want to enhance this since this code is from CM deployment shell script